### PR TITLE
Refactor APP config management

### PR DESCRIPTION
**Pull request by hchataing**
- Move enum definitions to uci_packets.pdl
- Move AppConfig declaration to separate module
  for readability
- Sync APP config definitions with UCI 2.0

### CODE REVIEW

File: py/pica/pica/packets/uci.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # File generated from <stdin>, with the command:
-#  ./pdl-compiler/scripts/generate_python_backend.py
+#  /home/henrichataing/Projects/github/google/pdl/pdl-compiler/scripts/generate_python_backend.py
```
Comment (SilverBzH) [inline py/pica/pica/packets/uci.py:2]: Does this path has to be here ?

File: src/lib.rs [hunk 2]
```diff
@@ -412,7 +413,7 @@ impl Pica {
         let mut measurements = Vec::new();
 
         // Look for compatible anchors.
-        for mac_address in session.get_dst_mac_addresses() {
```
Comment (SilverBzH) [inline src/lib.rs:415]: Is there a specific reason to make it singular ?

File: src/app_config.rs [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+    fn default() -> Self {
+        AppConfig {
+            device_type: None,
+            ranging_round_usage: None,
+            sts_config: uci::StsConfig::Static,
+            multi_node_mode: None,
+            channel_number: uci::ChannelNumber::ChannelNumber9,
+            number_of_controlees: 1,
+            device_mac_address: None,
+            dst_mac_address: vec![],
```
Comment (SilverBzH) [inline src/app_config.rs:94]: Is it singular in the specification ?

Comment (hchataing) [reply src/app_config.rs:94]: Yes, because of the default use case, which is to give the peer address in one to one mode

Comment (SilverBzH) [reply src/app_config.rs:94]: I see, understood

File: py/pica/pica/packets/uci.py (hunk 1 above)
Comment (hchataing) [reply py/pica/pica/packets/uci.py:2]: Nope

**Review by SilverBzH — APPROVED** [review]

**Pull request merged**